openmetadata-ui/src/main/resources/ui/src/pages/EntityVersionPage/EntityVersionPage.component.tsx

```diff
@@ -381,8 +376,8 @@ const EntityVersionPage: FunctionComponent = () => {
                       : undefined,
                   },
                   {
-                    name: database.name,
-                    url: getDatabaseDetailsPath(database.fullyQualifiedName),
+                    name: getPartialNameFromFQN(database.name, ['database']),
+                    url: getDatabaseDetailsPath(database.name),
```

Comment:
Same here

Reply:
Because from backend we are not getting `fullyQualifiedName` and in `name` field we are getting fqn value.